### PR TITLE
Child model & router/Pages table functionality

### PR DESCRIPTION
**Pull request by Aaron-TheCreator**
# Description

- What work was done?
- Adds functions to `childModel` and new endpoints to `childRouter`
- Why was this work done?
- needed to access table added in #84  , replaces `Drawing` & `Writing` 
- What feature / user story is it for?
- https://trello.com/c/dyH0YDuv/645-add-pages-to-submissions-record
- Any relevant links or images / screenshots
https://www.loom.com/share/aa54729ff1f046ecb2a7e7fa689cdefe

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge

## Has This Been Tested

- [X] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: api/child/childModel.js [hunk 1]
```diff
@@ -116,11 +116,14 @@ const getComplexityList = (ChildID) => {
 };
 
 //Will return a single submission
-const getSubmissionBySubId = async (id) => {
-  const foundSubmission = await db('Submissions-New').where({
-    id,
+const getSubmissionBySubId = async (childId, id) => {
```
Comment (tonysorensen) [inline api/child/childModel.js:119]: Really appreciated your help troubleshooting this. I relearned a lot.

File: api/child/childRouter.js [hunk 2]
```diff
@@ -440,22 +440,25 @@ router.get('/:id/submissions', authRequired, async (req, res) => {
 });
 
 //To return a single submission
-router.get('/:id/submissions/:id', authRequired, async (req, res) => {
-  //Pull submission ID out of URL parameter
-  const { id } = req.params;
+router.get('/:childId/submissions/:id', authRequired, async (req, res) => {
```
Comment (tonysorensen) [inline api/child/childRouter.js:443]: Again, we had to troubleshoot this one and I really appreciate the methodical way you approach problems.

**Review by tonysorensen — APPROVED** [review]
Good, clean, understandable code. This will help those coming behind us a lot.

File: api/child/childModel.js (hunk 1 above)
Comment (Aaron-TheCreator) [reply api/child/childModel.js:119]: Thank you, sir. JavaScript is fun.

**Review by ashtilawat — APPROVED** [review]

**Pull request merged**